### ISSUE TITLE
sentinelone.py returns "[Errno 2] No such file or directory"

### ISSUE THREAD

**Issue by paulashbrook** [munkireport/sentinelone#10]
Hello,
Using the latest sentinelone module for MunkiReport, munkireport-runner reports the following messages on my M1 MB Air running the latest macOS Ventura:
```
Running sentinelone.py
WARNING: sentinelone.py: [Errno 2] No such file or directory
```
and later
`WARNING: Can't open /usr/local/munkireport/scripts/cache/sentinelone.plist`

All other modules that I have enabled are working fine.

Please advise how I can gather more verbose detail from running the module.

**Comment by tuxudo** [comment]
What happens when you run `sudo /usr/local/munkireport/scripts/sentinelone.py` on that machine? Which version of the module are you using?

**Comment by paulashbrook** [comment]
Hello,
My MunkiReport-php install reported version 6.0.3, and also reported that no update was available.  However, I now see that version 2.0 was released **after** 6.03.
I've now 'upgraded' to 2.0 and can report that the module now works fine and that data from SentinelOne client installs is being reported.
This issue can be closed, thanks for the steer about versions.
